src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,63 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "USERS")
+public class User {
+
+    private Long id;
+    private String userName;
+    private String status;
```

Comment:
Jakie wartości będzie przyjmować pole `status`?

Reply:
Przyznam  że nie wiem - zgodnie z json ze specyfikacji -  "status": "1" - uznałem zatem że jest to pole wartościami typu String

Reply:
Pull request został zaktualizowany o zmodyfikowaną implementację klasy User, typ pola user tej klasy zmieniłem z String na Boolean.